ui/src/lib/types.ts

```diff
@@ -71,7 +71,7 @@ export type PublicationCreationStoreType = {
     ethicalStatementFreeText: string | null;
     updateEthicalStatementFreeText: (ethicalStatementFreeText: string | null) => void;
     updateEthicalStatement: (ethicalStatement: string) => void;
-    dataAccessStatement: string | null;
+    dataAccessStatement: string;
```

Comment:
again this is not related to the fix (and is already merged, I find github a bit strange in that respect bitbucket would show this as either not a change or a conflict)